### ISSUE TITLE
Fetch multiple events for large block ranges using goroutines/channels

### ISSUE THREAD

**Issue by bjartek** [onflow/flow-cli#316]
## Instructions

I have some code in go-with-the-flow https://github.com/bjartek/go-with-the-flow/blob/master/gwtf/event.go#L247 that I am pondering changing and submitting to this cli project. The idea is that it should be possible to specify several events and a large block size and partition the rangeQueries and run them in paralell. 

### Issue To Be Solved

Fetching two or more events for a large block size is currently not possible. 

### (Optional): Suggest A Solution

Include the go-with-the-flow channel/go-routine code into flow cli and exapand on the `flow events get` methods or possible introduct a `flow events get-many` command

**Comment by bjartek** [comment]
Obviously I want to the remove the filtering of ignored fields as that is not something that is needed in the flow-cli I think.

**Comment by sideninja** [comment]
I like the idea of fetching blocks in parallel. This would greatly increase the speed.

**Comment by bluesign** [comment]
This is a nice idea, but somehow I think it is better implemented with https://github.com/onflow/flow-go/issues/961 

Currently, flow-go caching with LRU, with very small cache size, those requests can evict all recent blocks from cache.

**Comment by simonhf** [comment]
I know this is closed but please note that there is an exciting PR [1] in the pipeline which will likely replace the existing LRU.

An issue with the existing LRU is that the bigger it is, the more Golang heap objects there are, and the slower GC is because GC loops through all heap objects even if it finally ends up pruning even a smaller number of heap objects. That's a reason why the LRU cache is relatively small. However, the new PR allows the LRU mechanism to scale without scaling the number of heap objects... :-)

[1] https://github.com/onflow/flow-go/pull/1778
